src/schema/v2/user.ts

```diff
@@ -256,6 +256,11 @@ export const UserType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLBoolean,
         resolve: async ({ data_transfer_opt_out }) => data_transfer_opt_out,
       },
+      identityVerified: {
```

Comment:
#minor but we _sometimes_ (v inconsistently) prefix boolean fields with `is`, like `isIdentityVerified`.